CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
+
+v1.3.0 / 2018-02-07
```

Comment:
Please don't update the change log in PRs, I will write the change notes myself when I publish.